images/techdocs-cli/tests/test_image.py

```diff
@@ -0,0 +1,37 @@
+import docker
+import pytest
+from docker.models.images import Image
+
+
+@pytest.fixture(scope="session")
+def docker_client() -> docker.DockerClient:
+    return docker.DockerClient(base_url="unix:///var/run/docker.sock")
```

Comment:
I'm not sure it is good to hardcode the base url for docker, on my computer it is in a different place, and I think this usually comes from an environment variable if not set here, and the environment should be setup correctly.

Reply:
I reused what we had done in the pypoetry.

Reply:
Fair enough, lets leave it like this, if it causes a problem for anyone we can deal with it then.